### PR TITLE
Adds poll command (a second time because this one passes tests)

### PR DESCRIPTION
**Pull request by BenSegal855**
**Describe this PR and why it should be merged:**
This pull requests adds a simple poll command (with alias vote). Running the command will place the first argument at the top of the returned message in bold. It will then put the other arguments (anywhere from two the ten others) under the first with number emojis in front of them. The first arg is the question or topic to be voted on. The next are the options that can be chosen. After sending the message, reactions are then added for the appropriate number of options.
**Status**
- [x] This PR has been tested and complies with this project's ESLint rules
- [x] This PR adds/modifes a command
- [x] The `examples` and `helpUsage` options have been created/updated accordingly

**Semantic Versioning**
- [x] This PR adds features for users
- [ ] This PR features bug fixes, or only style changes/refactorings, or no code changes
- [ ] This PR switches this project to a new a new Discord API library/new Discord.js framework

### CODE REVIEW

File: src/commands/Fun/Random/poll.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['vote'],
+			description: `Have ${NAME} create a poll for you`,
+			extendedHelp: 'This command requires you to have at least two options and no more than ten.',
+			examples: ['poll Is butt legs?|Butt is legs|Butt is butt'],
+			usage: '<question:string> <option1:string> <option2:string> [...]',
+			helpUsage: 'question | option 1 | option 2 | ...'
+		});
+	}
```
Comment (jwford) [inline src/commands/Fun/Random/poll.ts:17]: the description and extended help need to be internationalized, and helpUsage is unnecessary now. you can look at other commands and the en-US.ts language file to see how that should be done

File: src/commands/Fun/Random/poll.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+	public async run(msg: KlasaMessage, choices: string[]): Promise<Message> {
+		if (choices.length > 11) {
+			return msg.channel.send(`You have too many choices! The max is 10!`);
```
Comment (jwford) [inline src/commands/Fun/Random/poll.ts:21]: also needs to be internationalized

**Review by jwford — CHANGES_REQUESTED** [review]
i guess my main concern is that if we're going to have a poll comand, i'd like some way to for him to end the poll and tally the votes and announce a winner as well.

**Comment by BenSegal855** [comment]
> i guess my main concern is that if we're going to have a poll comand, i'd like some way to for him to end the poll and tally the votes and announce a winner as well.

Would you want some kind of configurable length before the poll ends or just an arbitrary five or so minutes for every poll?

**Comment by jwford** [comment]
> > i guess my main concern is that if we're going to have a poll comand, i'd like some way to for him to end the poll and tally the votes and announce a winner as well.
> 
> Would you want some kind of configurable length before the poll ends or just an arbitrary five or so minutes for every poll?

configurable, preferably

**Comment by BenSegal855** [comment]
> > > i guess my main concern is that if we're going to have a poll comand, i'd like some way to for him to end the poll and tally the votes and announce a winner as well.
> 
> > 
> 
> > Would you want some kind of configurable length before the poll ends or just an arbitrary five or so minutes for every poll?
> 
> 
> 
> configurable, preferably

So command syntax being something like `s;poll duration|question|choice1|choice2|...` and than after the duration a message would be sent saying something along these lines `choice1 has won question with x votes`

**Comment by BenSegal855** [comment]
Closing for now and I will be re-doing this code pretty much from the ground up Soon:tm:

**Pull request closed without merging**